packages/mds-agency/tests/integration-tests.spec.ts

```diff
@@ -1537,6 +1552,81 @@ describe('Tests for taxi modality', async () => {
   }
 })
 
+describe('Tests for tnc modality', async () => {
+  before(async () => {
+    await Promise.all([db.reinitialize()])
+  })
+
+  it('verifies post tnc success', done => {
+    request
+      .post(pathPrefix('/vehicles'))
+      .set('Authorization', AUTH)
+      .send(TEST_TNC)
+      .expect(201)
+      .end((err, result) => {
+        done(err)
+      })
+  })
+
+  for (const tncEvent of TNC_VEHICLE_EVENT) {
+    const validStates = TNC_EVENT_STATES_MAP[tncEvent]
+    for (const vehicle_state of validStates) {
+      it(`verifies ${tncEvent} success`, done => {
+        const { device_id } = TEST_TNC
+        const body = {
+          event_types: [tncEvent],
+          vehicle_state,
+          telemetry: TEST_TELEMETRY,
+          timestamp: now(),
+          ...(tncEvent.startsWith('trip_')
+            ? { trip_id: '1f943d59-ccc9-4d91-b6e2-0c5e771cbc6b', trip_state: vehicle_state as TRIP_STATE }
+            : {})
+        }
+        request
+          .post(pathPrefix(`/vehicles/${device_id}/event`))
+          .set('Authorization', AUTH)
+          .send(body)
+          .expect(201)
+          .end((err, result) => {
+            // test.string(result.body.status).is(EVENT_STATUS_MAP[taxiEvent])
```

Comment:
Did you mean to comment this out? Looks like a legit test

Reply:
I think I'd commented it out in Taxi, not entirely sure why I did.. will try uncommenting and seeing if it breaks anything :)